modules/ecs6-class/line.js

```diff
@@ -1,15 +1,18 @@
 const Point = require("./point");
 
 class Line {
-    constructor({ point1 = new Point(), point2 = new Point(), n = undefined, slope = undefined }) {
+    constructor({ point1 = new Point(), point2 = new Point(), n = undefined, slope = undefined }) {  
         this.point1 = point1;
```

Comment:
you have to add more checks in the function
what happens when point is an array, what happens when n is a string etc...